### PR TITLE
Add beacon2-ri-tools

### PR DESCRIPTION
**Pull request by khaled196**
Describe your pull request here

----

Please read the [guidelines for Bioconda recipes](https://bioconda.github.io/contributor/guidelines.html) before opening a pull request (PR).

* If this PR adds or updates a recipe, use "Add" or "Update" appropriately as the first word in its title.
* New recipes not directly relevant to the biological sciences need to be submitted to the [conda-forge channel](https://conda-forge.org/docs/) instead of Bioconda.
* PRs require reviews prior to being merged. Once your PR is passing tests and ready to be merged, please issue the `@BiocondaBot please add label` command.
* Please post questions [on Gitter](https://gitter.im/bioconda/Lobby) or ping `@bioconda/core` in a comment.

<details>
  <summary>Please use the following BiocondaBot commands:</summary>

Everyone has access to the following BiocondaBot commands, which can be given in a comment:

<table>
  <tr>
    <td><code>@BiocondaBot please update</code></td>
    <td>Merge the master branch into a PR.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please add label</code></td>
    <td>Add the <code>please review & merge</code> label.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please fetch artifacts</code></td>
    <td>Post links to CI-built packages/containers. <br />You can use this to test packages locally.</td>
  </tr>
</table>

For members of the Bioconda project, the following command is also available:

<table>
  <tr>
    <td><code>@BiocondaBot please merge</code></td>
    <td>Upload built packages/containers and merge a PR. <br />Someone must approve a PR first! <br />This reduces CI build time by reusing built artifacts.</td>
  </tr>
</table>

Also, the bot watches for comments from non-members that include `@bioconda/<team>` and will automatically re-post them to notify the addressed `<team>`.

</details>

### CODE REVIEW

**Comment by bgruening** [comment]
@khaled196 I guess we really need the dependency graph. Which PR depends on which?

**Comment by khaled196** [comment]
@bgruening 

Thank you for looking into this, I will make the graph and send it first thing in the morning

**Comment by khaled196** [comment]
> @khaled196 I guess we need the dependency graph. Which PR depends on which?

Hi @bgruening, Kindly find the link to the docs with the dependence graph. I know it is not the best way to show it. However, you can directly access the pull request for the dependences in this method. 

The dependencies start from from order the top till the end

File: recipes/beacon2-ri-tools/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+    - perl-json-validator
+test:
+  # Perl 'use' tests 
+  commands:
+    - beacon -h
+
+about:
+  home: https://github.com/EGA-archive/beacon2-ri-tools/tree/main
+  license: GPL-3.0-only
+  summary: "Script to parse a VCF having SnepEff/SnpSift annotations"
```
Comment (bgruening) [inline recipes/beacon2-ri-tools/meta.yaml:73]: Is that a correct summary?

Comment (bgruening) [reply recipes/beacon2-ri-tools/meta.yaml:73]: It sounds like overkill. I'm not sure why someone writes this in perl nowadays. Lets get started with the VCF mode and ask the beacon overlords if this is going to stay and we are supposed to carry those perl libs around forever.

Comment (khaled196) [reply recipes/beacon2-ri-tools/meta.yaml:73]: I will leave MongoDB for later then. 

about the description how is "Beacon v2 Reference Implementation (Data ingestion tools)" sound?

**Comment by bgruening** [comment]
Thanks a lot!

**Pull request merged**